pkgs/development/python-modules/pymd4c/default.nix

```
@@ -0,0 +1,26 @@
+{ stdenv
+, pkgs
+, buildPythonPackage
+, fetchPypi
+, python3Packages
+, md4c
+}:
+
+buildPythonPackage rec {
+  pname = "pymd4c";
+  version = "0.4.6.0b1";
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "07s3arn85ri92im6x3ipljdmrxmpik7irs06i6lm17j1x6j9841d";
+  };
+  nativeBuildInputs = [ pkgs.pkgconfig ];
```

for new usage of pkgconfig, we prefer the name over the alias
```suggestion
  nativeBuildInputs = [ pkg-config ];
```